### PR TITLE
tracy: add 3 options introduced in 0.11.0

### PR DESCRIPTION
**Pull request by toge**
### Summary
Changes to recipe:  **tracy/***

#### Motivation
This recipe has unsupported options introduced in 0.11.0.
This PR tries to add these.

#### Details
https://github.com/conan-io/conan-center-index/pull/24647#issuecomment-2242493081

- `TRACY_LIBUNWIND_BACKTRACE` has bug which has been solved in upstream.
- Because `TRACY_DEMANGLE` is an experimental options, I think it is not time to add it.

---
- [x] Read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md)
- [x] Checked that this PR is not a duplicate: [list of PRs by recipe](https://github.com/conan-io/conan-center-index/discussions/24240)
- [x] Tested locally with at least one configuration using a recent version of Conan

### CODE REVIEW

**Review by perseoGI — APPROVED** [review]
LGTM

File: recipes/tracy/all/conanfile.py [hunk 1]
```diff
@@ -79,6 +92,10 @@ def validate(self):
         if self.info.settings.compiler.get_safe("cppstd"):
             check_min_cppstd(self, 11)
 
+        # libunwind_backtrace is not supported in 0.11.0. https://github.com/wolfpld/tracy/pull/841
+        if self.options.get_safe("libunwind_backtrace"):
+            raise ConanInvalidConfiguration(f"libunwind_backtrace is not supported in {self.ref}")
```
Comment (AbrilRBS) [inline recipes/tracy/all/conanfile.py:97]: Should this not have an extra check for `Version(self.version) == "0.11.0"`?

**Review by AbrilRBS — APPROVED** [review]
Thanks!

**Review by perseoGI — APPROVED** [review]
LGTM

**Pull request merged**